js/components/src/livestream-provider/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useRef } from "react";
+import { LivestreamContext, makeLivestreamStore } from "../livestream-store";
+import { useLivestreamWebsocket } from "./websocket";
+
+export function LivestreamProvider({
+  children,
+  src,
+}: {
+  children: React.ReactNode;
+  src: string;
+}) {
+  const store = useRef(makeLivestreamStore()).current;
+  (window as any).livestreamStore = store;
```

Comment:
Ummmmm. Good for debugging. Prolly bad practice in general though. Does this even work on native?